Tests/AudioKitTests/Node Tests/Player Tests/AudioPlayerTests.swift

```diff
@@ -2,6 +2,66 @@ import AudioKit
 import AVFoundation
 import XCTest
 
+final class AudioEngineTwo {
+
+    private let engine = AudioEngine()
+
+    let mixer: Mixer
+
+    let player = AudioPlayer()
+
+    init() {
+        mixer = Mixer()
+        mixer.addInput(player)
+        engine.output = mixer
+
+        do {
+            try Settings.setSession(category: .playAndRecord,
```

Comment:
Weird, this compiles in Xcode. Not sure why it doesn't compile here. Could possibly be related to this? https://stackoverflow.com/questions/67494326/swiftui-cannot-infer-contextual-base-in-reference-to-member